webapp/portlet/src/main/webapp/vue-apps/activity-composer-app/components/ExoActivityComposer.vue

```diff
@@ -369,7 +369,7 @@ export default {
     },
     cleanInitialActivityBody: function()  {
       const div = document.createElement('div');
-      div.innerHTML = this.activityBody;
+      div.innerHTML = this.message;
```

Comment:
@ahmadmido I have fixed it in this PR : https://github.com/Meeds-io/social/pull/899/commits/74dbbe653411cf2244846eb057cdb7eca4d28cdf
You can close this one, thanks.